liquibase-standard/src/main/java/liquibase/change/core/LoadDataChange.java

```diff
@@ -350,21 +350,17 @@ public SqlStatement[] generateStatements(Database database) {
                         } else if (columnConfig.getType().equalsIgnoreCase("date")
                                 || columnConfig.getType().equalsIgnoreCase("datetime")
                                 || columnConfig.getType().equalsIgnoreCase("time")) {
-                            if ("NULL".equalsIgnoreCase(value) || "".equals(value)) {
```

Comment:
Initially, the purpose of updating this line was just to replace`"".equals(value)` with `value.isEmpty()` doing this I saw there were some tests that were failing because this was, don't remember exactly if it was when having a `null` or when having an empty string (`''`), so I changed the logic this way, but I think I missed the case of when the value (date) is `null` then set value as null (as it was before). I'm going to update this.

Reply:
I see.. do we have tests covering that?

Reply:
I updated/refactored some tests that are testing the updated code.